compiler/src/transform/filter-by-availability.ts

```diff
@@ -257,8 +258,8 @@ async function filterSchema(inPath: string, outPath: string, stack: boolean, ser
 
 const inputPath: string = argv.input ?? join(__dirname, '..', '..', '..', 'output', 'schema', 'schema.json')
 const outputPath = argv.output ?? join(__dirname, '..', '..', '..', 'output', 'schema', 'schema-filtered.json')
-const stack = !!argv.stack
-const serverless = !!argv.serverless
+const stack: boolean = (argv.stack !== undefined)
```

Comment:
This makes it more strict, which I assume is what you want. `!!argv.stack` would return `false` if its value was `undefined`, `0`, `""`, `null`, `-0` (don't get me started :roll_eyes:), and [a few others](https://developer.mozilla.org/en-US/docs/Glossary/Falsy). Your change will now return `true` for all but `undefined`.

Reply:
Just realized I must've been commenting on a single commit where you changed your own work in progress. :facepalm:

Reply:
You did, that's fined, the code is the same in the end. :)